res/values/strings.xml

```diff
@@ -205,6 +210,9 @@
     <string name="media_rewind_description">Rewind button</string>
     <string name="media_play_pause_description">Play or pause button</string>
     <string name="media_forward_description">Fast forward button</string>
+    <string name="button_text_camera"> Camera </string>
+    <string name="upload_from_camera_title"> Upload From Camera </string>
+    <string name="capture_button_text"> Capture </string>
```

Comment:
Not used?

Reply:
capture_button_text seems not to be used either.

Reply:
Yes I just saw that. I used it when I was not using bottom sheets and instead using a FAB.